scm/webhook.go

```diff
@@ -37,6 +37,14 @@ type (
 		Commits []Commit
 	}
 
+	// PipelineHook
+	PipelineHook struct {
+		Repo     Repository
+		Commit   Commit
+		Pipeline Pipeline
+		User     User
```

Comment:
In other webhooks, we refer to the user as the `Sender` (user that caused the hook to get sent)